### PR TITLE
NAS-110177 / 12.0 / Non-invasive linter rules

### PR DESCRIPTION
**Pull request by undsoft**

### CODE REVIEW

**Comment by denysbutenko** [comment]
There's merge conflict

**Comment by undsoft** [comment]
@denysbutenko 
> There's merge conflict

Fixed.

File: src/app/pages/task-calendar/replication/replication-list/replication-list.component.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ import helptext from '../../../../helptext/task-calendar/replication/replication
 
 @Component({
   selector: 'app-replication-list',
-  template: '<entity-table [title]="title" [conf]="this"></entity-table>',
+  template: '<entity-table [title]=\'title\' [conf]=\'this\'></entity-table>',
```
Comment (denysbutenko) [inline src/app/pages/task-calendar/replication/replication-list/replication-list.component.ts:17]: This line looks weird. Maybe we could enable avoidEscape for quotes? https://eslint.org/docs/rules/quotes#avoidescape

Comment (undsoft) [reply src/app/pages/task-calendar/replication/replication-list/replication-list.component.ts]: Agreed, also breaks message extraction. As for avoidEscape, it's already on, so not really sure what happened here.

**Review by denysbutenko — APPROVED** [review]
LGTM

**Pull request merged**